### PR TITLE
Move electrs startscript to tempdir and fix nits

### LINKED ISSUE #86: electrs breaks when the datadir variable is set
I set the `electrs.dataDir` variable to some other location and I'm getting these logs in electrs:
```
Jul 10 08:55:55 dasilva-nix s4j86k2qkxz8473v1gy7xh3q5jfxp743-unit-script-electrs-pre-start[1425]: chmod: cannot access '/media/bitcoin-extra/electrs/startscript.sh': No such file or directory
```

So I suppose this startscript.sh script is somehow hard-coded into a path, but then the datadir is used to invoke it. Why isn't that script really hardcoded and just put in some hardcoded location?

### PR DESCRIPTION
**Pull request by nixbitcoin**
This PR fixes #86 by placing startscript in a systemd RuntimeDirectory only accessible by the electrs user as per @jonasnick's suggestion. It also sets a cfg.user and cfg.group variable and removes the hardcoded electrs user and group in several places.

I'm a bit concerned that I can't pass `${RuntimeDirectory}` from `serviceConfig` to `preStart` and had to hardcode `/run/electrs/startscript.sh`, but I've seen this done in other nix modules upstream. Will this be an issue?

Closes #93

### CODE REVIEW

File: modules/electrs.nix [hunk 1]
```diff
@@ -66,13 +77,14 @@ in {
       preStart = ''
         mkdir -m 0770 -p ${cfg.dataDir}
         chown 'electrs:electrs' ${cfg.dataDir}
```
Comment (jonasnick) [inline modules/electrs.nix]: Can you remove that? Seems even less useful now?

Comment (jonasnick) [reply modules/electrs.nix]: Perhaps also move the `chown -R '${cfg.user}:${cfg.group}' ${cfg.dataDir}` right below `mkdir`.

**Comment by nixbitcoin** [comment]
Do you think `/run/electrs/startscript.sh` could cause problems?

**Comment by jonasnick** [comment]
@nixbitcoin No. What kind of problems are you thinking of?

**Comment by nixbitcoin** [comment]
> @nixbitcoin No. What kind of problems are you thinking of?

If somebody were to change RuntimeDirectory, startscript would no longer be located at `/run/electrs/startscript.sh`. However, I see no reason for a user doing this.

**Pull request merged**